src/Extension/Attributes/Util/AttributesHelper.php

```diff
@@ -72,6 +72,10 @@ public static function parseAttributes(Cursor $cursor): array
                 continue;
             }
 
+            if (\strpos($attribute, '=') === false) {
+                $attribute .= '=""';
+            }
+
             /** @psalm-suppress PossiblyUndefinedArrayOffset */
             [$name, $value] = \explode('=', $attribute, 2);
```

Comment:
The code below this point assumes we have a string value.  But if no value is given, we can skip that whole section with something like this:

```php
$parts = \explode('=', $attribute, 2);
if (\count($parts) === 1) {
    $attributes[$attribute] = true;
    continue;
}

[$name, $value] = $parts;
```